### DISCUSSION TITLE
[User Poll] Internationalization (i18n) support

### DISCUSSION THREAD

**Discussion by dimitropoulos** [Kong/insomnia#3497] [Ideas]
I thought it might be nice to use this discussion as a place to organize which languages are in the highest demand.

If you'd like to see one language in particular, upvote a comment corresponding to that language below.

To be clear, when I say `upvote` I'm talking about this up arrow:
![Screenshot_20210622_111952](https://user-images.githubusercontent.com/15232461/122952068-c53c3f80-d34b-11eb-840d-5f8f15d658e2.png)

(and not the :+1: emoji we are all accustomed to on github issues)

 If you'd like to add a new one, please do so in a new comment.
 
 Here's a good wikipedia list https://en.wikipedia.org/wiki/List_of_languages_by_number_of_native_speakers

**Comment by dimitropoulos** [comment]
Spanish (Español)

**Comment by dimitropoulos** [comment]
Italian (Italiano)

**Comment by dimitropoulos** [comment]
German (Deutsch)

**Comment by dimitropoulos** [comment]
French (Français)

**Comment by dimitropoulos** [comment]
Korean (한국어)

**Comment by dimitropoulos** [comment]
Japanese (日本語)

**Comment by dimitropoulos** [comment]
Simplified Chinese (简体中文)

**Reply by night2049** [reply]
how long add the language support

**Reply by SuSuSoo** [reply]
As of 2024, how has the translation project advanced? The project has captured the interest and promotion of various self-media in China, and it is expected that the launch of a Simplified Chinese version in the near future will provide Chinese users with an improved experience.

**Reply by GaoJuqian** [reply]
Can we open the development of language plug-ins?

**Reply by jimie-chaos** [reply]
Do you have a plan?

**Comment by dimitropoulos** [comment]
Traditional Chinese (繁体中文)

**Comment by dimitropoulos** [comment]
Hindi (हिन्दी)

**Comment by dimitropoulos** [comment]
Russian (русский язык)

**Comment by dimitropoulos** [comment]
Portuguese (português)

**Comment by dimitropoulos** [comment]
English (UK)

**Reply by dimitropoulos** [reply]
I actually think this should be the first language we support: for the simple reason that it will be the easiest to do since it's closest to the current language and also because it's likely to be the easiest to test as we figure out our strategy for multi language support.
